Specs/Scene/ExpressionSpec.js

```diff
@@ -803,15 +808,10 @@ defineSuite([
         expect(expression.evaluate(undefined)).toEqual(/a/mg);
     });
 
-    it('throws if regex constructor does not have literal as argument', function() {
-        expect(function() {
-            return new Expression(new MockStyleEngine(), 'regExp(1+1)');
-        }).toThrowDeveloperError();
-    });
-
     it('throws if regex constructor has invalid flags', function() {
         expect(function() {
-            return new Expression(new MockStyleEngine(), 'regExp("a", "q")');
+            var expression = new Expression(new MockStyleEngine(), 'regExp("a", "q")');
+            expression.evaluate(undefined);
```

Comment:
`evaluate` is expected to throw, right?  Just put that in the `expect` function.